packages/synapse-react-client/src/components/EntityFinder/tree/VirtualizedTree.tsx

```diff
@@ -363,11 +372,11 @@ export function Node(
         </div>
       )}
 
-      <div className="EntityName" ref={ref}>
-        <Tooltip title={tooltipContent} placement="right">
+      <Tooltip title={tooltipContent} placement="right">
+        <div className="EntityName" ref={ref}>
           {nodeText}
-        </Tooltip>
-      </div>
+        </div>
+      </Tooltip>
```

Comment:
Wrap tooltip around the div so that it is positioned after the visible text, not the end of the overflowing & not visible text.